documentation/airport-format.md

```diff
@@ -1,309 +1,228 @@
-+-------------+---+------------------------------------+-------+
-| 'offset'    |   | min into the cycle to start at     | 0     |
-+-------------+---+------------------------------------+-------+
-| 'period'    |   | length of each cycle, in minutes   | 30    |
-+-------------+---+------------------------------------+-------+
-| 'variation' | * | the amount to +/- from 'frequency' | 0     |
-+-------------+---+------------------------------------+-------+
-|     (also include "bare minimum parameters" - see above)     |
-+--------------------------------------------------------------+
-
-
-### Surge
-
-The wave algorithm generates a group of aircraft back to back. For departures
-the spacing is 10 seconds, for arrivals, you can specify the entrail distance
-while in and out of the "surge". This way, a "surge" can be gentle, or extreme,
-and at any arrival rate.
-
-Note that if you request something that's impossible to deliver, like either...
-   - "frequency": 50, "entrail": [ 7, 15] <-- even if all are 7MIT, that's 35acph
-   - "frequency": 7,  "entrail": [10, 25] <-- even if all are 25MIT, that's 10acph
-   - Note: The above assumes spawn speed of 250kts, for example purposes
-...the game will throw a warning in the console advising you that it has
-clamped the arrival rate to match the entrail and speed settings, and it tells
-you what range of frequencies it is mathematically capable of delivering.
-
-              PARAMETERS SPECIFIC TO 'SURGE':
-  PARAMETER  REQ        PARAMETER DESCRIPTION         DEFAULT
-+-----------+---+---------------------------------------------+-------+
-| 'offset'  |   | min into the cycle to start at              | 0     |
-+-----------+---+---------------------------------------------+-------+
-| 'period'  |   | length of each cycle, in minutes            | 30    |
-+-----------+---+---------------------------------------------+-------+
-|           |   | array of:                                   |       |
-| 'entrail' | * | [ miles between arrivals during the surge,  | [5.5, | <-- only for arrivals
-|           |   |   miles between arrivals during the lull  ] |  10]  |
-+-----------+---+---------------------------------------------+-------+
-|        (also include "bare minimum parameters" - see above)         |
-+---------------------------------------------------------------------+
-
-
-
-Aircraft/Airline selectors
---------------------------
-
-Both departure and arrival blocks specify a weighted list of airlines
-to use for that block.  The airline code may be optionally followed by
-a particular fleet for that operator.
-
-Example:
-```
-"airlines": [
-  ["BAW", 10],
-  ["AAL/long", 2]
-]
+#### `spawnPatterns` Build Process
 
-Select an aircraft from BAW's (British Airways) default fleet five
-times as often as an aircraft is selected from AAL's (American
-Airlines) long haul fleet.
+See [spawnPatternReadme.md](src\assets\scripts\client\trafficGenerator\spawnPatternReadme.md)
```

Comment:
we should probably move this doc to the documentation folder for easier reference.